File: app/app/src/main/java/com/example/n_ai_tupeu/database/ChallengeType.java
package com.example.n_ai_tupeu.database;

public class ChallengeType {
    public enum Type {
        Dare(1),
        Truth(0);

        private final int value;

        Type(int value) {
            this.value = value;
        }

        public int getValue() {
            return value;
        }
    }


}

File: GameAPI/GameAPI/Models/ChallengeType.cs
using Newtonsoft.Json.Converters;
using System.Text.Json.Serialization;

namespace GameAPI.Models
{

    public class ChallengeType
    {
        //[JsonConverter(typeof(StringEnumConverter))]
        public enum Type
        {
            Truth,
            Dare
        }

    }
}
